pkg/neorpc/errors.go

```diff
@@ -138,6 +138,18 @@ const (
 	ErrExecutionFailedCode = -608
 )
 
+var (
+	// ErrCompatGeneric is an error returned by nodes not compliant with the neo-project/proposals#156
+	// (NeoGo pre-0.102.0 and all known C# versions).
+	// It can be returned for any call and doesn't have any specific meaning.
+	// Deprecated: to be removed after all nodes adopt new error standard.
```

Comment:
IIRC it's important to have a blank line before `deprecated:`, otherwise it's the same paragraph.

Reply:
I didn't catch that. Like this: 
	// ErrCompatGeneric is an error returned by nodes not compliant with the neo-project/proposals#156
	// (NeoGo pre-0.102.0 and all known C# versions).
	// It can be returned for any call and doesn't have any specific meaning.
	//
	// Deprecated: to be removed after all nodes adopt new error standard.
	ErrCompatGeneric = NewErrorWithCode(-100, "RPC error")
?

Reply:
fixed